share/models/ingest.py

```diff
@@ -132,10 +132,18 @@ def natural_key(self):
         return (self.label,)
 
     def get_harvester(self, **kwargs):
-        return self.harvester.get_class()(self, **{**kwargs, **(self.harvester_kwargs or {})})
+        """Return a harvester instance configured for this SourceConfig.
+
+        Passed kwargs override or add to harvester_kwargs.
+        """
+        return self.harvester.get_class()(self, **{**(self.harvester_kwargs or {}), **kwargs})
```

Comment:
I'm not sure how I feel about passing options here rather to the fetch/harvest methods. I think it make sense to have both.
`set_spec` feels like it would be more appropriate as a per call parameter rather than a per instance parameter.